src/MovieContainer.js

```diff
@@ -18,12 +18,26 @@ const MovieContainer = ({movies, changeSection}) => {
     )
   })
 
+  const filteredMovieCards = filteredMovies.map(movie => {
+    return (
+      <Poster 
+        id={movie.id}
+        poster_path={movie.poster_path}
+        backdrop_path={movie.backdrop_path}
+        title={movie.title}
+        average_rating={movie.average_rating}
+        release_date={movie.release_date}
+        key={movie.id}
+        changeSection={changeSection}
+      />
+    )
+  })
+
   return (
-    <div className='movie-container'>
-      {movieCards}
-    </div>
+      <div className='movie-container'>
+        {!filteredMovies.length ? movieCards : filteredMovieCards}
```

Comment:
Nice and clean way to alternate between normal movie array and filtered movie array